packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -289,19 +291,19 @@ const helpers = {
             pipe(
               previous,
               O.alt(() => {
-                logger.trace(`Unable to find a ${statusCodes[index]} response definition`);
+                logger.debug(`Unable to find a ${statusCodes[index]} response definition`);
```

Comment:
trace is not caught by our signals logger, so I moved it back to debug as it should be